installer/roles/kubernetes/templates/credentials.py.j2

```diff
@@ -12,14 +12,15 @@ DATABASES = {
         },
     }
 }
-BROKER_URL = 'amqp://{}:{}@{}:{}/{}'.format(
-    "{{ rabbitmq_user }}",
-    "{{ rabbitmq_password }}",
-    "localhost",
-    "5672",
-    "awx")
+
+BROKER_URL = 'redis://{}:{}/'.format(
+    "{{ kubernetes_redis_hostname }}",
+    "{{ kubernetes_redis_port }}",)
+
 CHANNEL_LAYERS = {
-    'default': {'BACKEND': 'asgi_amqp.AMQPChannelLayer',
-                'ROUTING': 'awx.main.routing.channel_routing',
-                'CONFIG': {'url': BROKER_URL}}
+    'default': {'BACKEND': 'channels_redis.core.RedisChannelLayer',
+                'CONFIG': {
+                    'hosts': [("{{ kubernetes_redis_hostname }}", {{ kubernetes_redis_port|int }})]
+                    'capacity': 10000,
```

Comment:
What's this `capacity` about?

Reply:
100 is the default capacity for a channel. If the client doesn't read
the socket fast enough, websocket messages can and will be lost. This
increases the default to 10,000. I figured that the callback receiver processes in 1,000 batches and we run 4 of them so roughly 2x + some wiggle room.

Reply:
The callback receiver uses a different queue. One that is fully under our control. That queue will just continue to grow, unbounded.